site/assets/scss/_masthead.scss

```diff
@@ -7,32 +7,70 @@
     line-height: 1;
   }
 
-  p:not(.lead) {
-    color: $gray-700;
-  }
-
   .btn {
     padding: .8rem 2rem;
     font-weight: 600;
   }
+
   .lead {
-    @include font-size(1.5rem);
+    @include font-size(1rem);
     font-weight: 400;
     color: $gray-700;
   }
+
+  .highlight {
+    padding: .5rem 4rem .5rem 1rem;
+    margin-bottom: 0;
+    line-height: 1.25;
+    @include border-radius(.5rem);
+  }
+  .btn-clipboard {
+    top: .5rem;
+  }
+
+  #carbonads { // stylelint-disable-line selector-max-id
+    margin-right: auto;
+    margin-left: auto;
+  }
```

Comment:
Could we imagine using utilities for such things? `.mx-auto` should be fine, isn't it? And there're quite a few declarations in site styles that we may drop. Could be tackled later, obviously.

Reply:
All the HTML is added dynamically, so I cannot modify it with utilities.